system/Session/Handlers/ArrayHandler.php

```diff
@@ -76,15 +76,15 @@ public function destroy($id): bool
     }
 
     /**
-     * Cleans up expired sessions.
-     *
-     * @param int $max_lifetime Sessions that have not updated
-     *                          for the last max_lifetime seconds will be removed.
-     *
-     * @return false|int Returns the number of deleted sessions on success, or false on failure.
-     */
+    * Cleans up expired sessions.
+    *
+    * @param int $maxLifetime Sessions that have not updated
+                            for the last max_lifetime seconds will be removed.
+    *
+    * @return false|int Returns the number of deleted sessions on success, or false on failure.
+    */
     #[ReturnTypeWillChange]
-    public function gc($max_lifetime)
```

Comment:
this is on purpose to have underscored variable, see

https://github.com/codeigniter4/CodeIgniter4/blob/efc1bca5f144a279b3aa52eb70158b5790494347/rector.php#L98-L101

please revert these changes on session handlers files.

Reply:
It isn't apparent to me why this PR should be touching Session at all - maybe working off an old branch, or somehow some files got crossed?

Reply:
I am not sure, probably direct run rector to specific file/dir on some os (windows ?) which probably somehow make skip config skipped. The run rector should be on root to ensure all applied with the files dependencies if any.

```bash
vendor/bin/rector --dry-run
```

Reply:
@samsonasik I recently tried running Rector latest version to try the parallel feature on my Windows using `vendor/bin/rector` and it seems these changes (changing underscore to camelCase, and removing method calls) occur. Not sure but it seems the root `rector.php` is not used?

Reply:
@paulbalandan I am not sure, could you try latest `"rector/rector": "dev-main"`? The latest dev-main solve the --config usage if used. If the issue persist in parallel only, probably in rector.php, it needs if os check to not run parallel on windows, 

It would be better to have issue open at rector repo with step to reproduce or github action repo sample that show the issue